Problems with signed commits

Hi, 

My commits are PGP signed.

HTML building failed with:

```
getting Git timestamps for source files... [  3%] ~/src/myProject/SubDir                                                 
Extension error (sphinx_last_updated_by_git):
Handler <function _env_updated at 0x7f928d993040> for event 'env-updated' threw an exception (exception: unexpected git output in ~/src/myProject/SubDir: b'gpg: Signature made So 17 Apr 2022 19:44:16 CEST\n')
make: *** [Makefile:57: html] Error 2
```

###  Workaround:

`git config log.showSignature false`

--- a/src/sphinx_last_updated_by_git.py
+++ b/src/sphinx_last_updated_by_git.py
@@ -48,7 +48,8 @@ def update_file_dates(git_dir, file_dates):
     process = subprocess.Popen(
         [
             'git', 'log', '--pretty=format:%n%at%x00%P', '--author-date-order',
-            '--relative', '--name-only', '-z', '-m', '--', *requested_files
+            '--relative', '--name-only', '--no-show-signature', '-z', '-m',
+            '--', *requested_files
         ],
         cwd=git_dir,
         stdout=subprocess.PIPE,

